qemu/tests/qemu_io_blkdebug.py

```diff
@@ -92,8 +92,8 @@ def run(test, params, env):
         else:
             session.close()
             try:
-                image_io.snapshot_del(blkdebug_cfg=blkdebug_cfg)
-                output = ""
+                cmd_result = image_io.snapshot_del(blkdebug_cfg=blkdebug_cfg)
+                output = cmd_result.stdout + cmd_result.stderr
```

Comment:
You said "As the function is called during negative test and function will not return is expected. So set the ignore_status to avoid raising a exception."  in avocado-framework/avocado-vt#553

But  Here try to catch CmdError. why?